webpack/dev.ts

```diff
@@ -44,7 +44,10 @@ export const getDevConfig = (target: Target): Configuration => {
         CONFIG_PERMISSION_HOSTS: JSON.stringify({
           origins: ['http://*/*', 'https://*/*']
         }),
-        CONFIG_ALLOWED_PROTOCOLS: JSON.stringify(['http:', 'https:'])
+        CONFIG_ALLOWED_PROTOCOLS: JSON.stringify(['http:', 'https:']),
+        CONFIG_OPEN_PAYMENTS_REDIRECT_URL: JSON.stringify(
+          'https://webmonetization.org/welcome'
```

Comment:
Should we stick to using `localhost:xyzz` one for dev?

Reply:
It should be okay for now. We change to a local version in the future.